cdap-ui/app/cdap/services/datasource/index.js

```diff
@@ -89,7 +89,7 @@ export default class Datasource {
         delete this.bindings[hash];
       } else {
         if (this.bindings[hash] && this.bindings[hash].type === 'POLL') {
-          this.bindings[hash].resource.interval = this.startClientPoll(hash, this.bindings[hash].resource.intervalTime);
+          this.bindings[hash].resource.interval = this.startClientPoll(hash, this.bindings, this.bindings[hash].resource.intervalTime);
```

Comment:
why do we have to pass in `this.bindings`? i think just passing `this.bindings[hash]` is good enough, or even just the hash

Reply:
We need access to bindings[resourceId].resource for the socket requests. @ajainarayanan is there any context I'm missing here?

Reply:
Actually you are right, we only need the hash. Have made changes accordingly!